src/views/Hero/Hero.scss

```diff
@@ -7,6 +7,7 @@
   min-height: 100%;
 
   &__homeView {
+    /*
```

Comment:
I'm wondering if we could just delete all these comments. If we no longer need these lines of code, let's just get rid of them!